.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+version: 2
+
+jobs:
+  "python-2.7": &test-template
```

Comment:
Not aware of this syntax '&test-template', you use it to refer to this job configuration for python36 and 37 so you can just override what you need?

Reply:
Yes, I took that from plotly/dash#222.